codalab/apps/teams/migrations/0004_auto_20220409_2242.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-04-09 22:42
+from __future__ import unicode_literals
+
+import codalab.azure_storage
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('teams', '0003_auto_20220328_1713'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='team',
+            name='image',
+            field=models.FileField(blank=True, null=True, storage=codalab.azure_storage.AzureStorage(account_key='qY4fuziQqP0N63JurVsEvp7rioAsqnwPdRRgpLKDoCTNrVj1Xwt0l/EJVP3Yxn2IySXqjgMFYnvWd+B1mgPUAA==', account_name='codalabpython3test', azure_container='public'), upload_to='team_logo', verbose_name='Logo'),
```

Comment:
On my side I'd like to look into the deconstruct method/decorator a bit to see if we can solve this before-hand.